cmd/cert-checker/main.go

```diff
@@ -118,13 +119,20 @@ func (c *certChecker) getCerts(unexpiredOnly bool) error {
 		args["now"] = c.clock.Now()
 	}
 
-	initialID, err := c.dbMap.SelectInt(
+	sni, err := c.dbMap.SelectNullInt(
 		"SELECT MIN(id) FROM certificates WHERE issued >= :issued AND expires >= :now",
 		args,
 	)
 	if err != nil {
 		return err
 	}
+	if !sni.Valid {
+		// a nil response was returned by the DB, so return error and fail
+		return fmt.Errorf("The SELECT query resulted in a NULL response from the DB.")
+	}
```

Comment:
Should be `errors.New()` since we're not passing args to be interpolated. You can commit this suggestion, but you'll also need to import `errors` before it will build correctly.

```suggestion
		return errors.New("the SELECT query resulted in a NULL response from the DB")
	}
```